eZ/Publish/Core/MVC/Symfony/Component/Tests/Serializer/Stubs/SerializerStub.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\Core\MVC\Symfony\Component\Tests\Serializer\Stubs;
```

Comment:
In my opinion, it looks more like Fake than Stub but I don't know that we want to stick strictly to definition ;)